doc/design/design_submit_job.md

```diff
@@ -0,0 +1,60 @@
+# Submit SQLFLow SQL Asynchronously
+
+## Abstract
+
+In the current system, SQLFlow client connects the SQLFlow server with long links. the SQLFlow client sends
+a request which contains the user-typed SQL and hung until the SQLFlow server finishes executing the SQL.
+Sometimes, it would cost a long time to execute the SQL and lead to gRPC time out.
```

Comment:
cost => take

Sometimes, the SQL job takes too much time and the gRPC calls timeout.

Reply:
Done.